config/environment.rb

```diff
@@ -4,3 +4,15 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Set up ActionMailer for SMTP
+
+ActionMailer::Base.smtp_settings = {
+  :address        => ENV["SMTP_SERVER"],
+  :domain         => ENV["EMAIL_DOMAIN"],
+  :port           => 587,
+  :user_name      => ENV["EMAIL_USERNAME"],
+  :password       => ENV["EMAIL_PASSWORD"],
+  :authentication => :plain,
```

Comment:
This can be
```ruby
ENV["SMTP_AUTH"].to_sym
```

and then in the docker-compose override, you can set `SMTP_AUTH=plain`